cmd/opentelemetry-prometheus-sidecar/main.go

```diff
@@ -200,12 +214,6 @@ func Main() bool {
 
 	logStartup(cfg, scfg.Logger)
```

Comment:
nit: please move logStart() so that it prints before calling selfTest(). I've been meaning to fix a similar problem, because the call to NewTailer() is already printing before logStartup(), so to make this Right, please move NewTailer() down and ensure that logStartup() is ordinarily the first line to print.

Reply:
I did this in a parallel PR.